src/util/getReferencedMessage.ts

```diff
@@ -0,0 +1,7 @@
+import { Message } from 'discord.js';
+
+export async function getReferencedMessage(msg: Message) {
+	if (!msg.reference?.messageID) return null;
+	const message = await msg.channel.messages.fetch(msg.reference.messageID);
+	return message;
+}
```

Comment:
Maybe rename this file `discordHelpers` or something like that? Later it might include a bunch of the module-agnostic functions currently living in the helpchan module. (Alternatively, don't create premature abstractions :P)

Reply:
I think it would be best to rename it when/if more things are moved into it